include/knowhere/expected.h

```diff
@@ -112,4 +112,20 @@ class expected {
 
 }  // namespace knowhere
 
+#define CHECK_STATUS(X)                         \
```

Comment:
Maybe change these to `KNOWHERE_CHECK_STATUS` to avoid definition pollution.

Reply:
ok